datarepo-clienttests/src/main/java/common/utils/MetricsUtils.java

```diff
@@ -0,0 +1,110 @@
+package common.utils;
+
+import com.google.api.gax.core.FixedCredentialsProvider;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.monitoring.v3.MetricServiceClient;
+import com.google.cloud.monitoring.v3.MetricServiceSettings;
+import com.google.monitoring.v3.Aggregation;
+import com.google.monitoring.v3.ListTimeSeriesRequest;
+import com.google.monitoring.v3.ProjectName;
+import com.google.monitoring.v3.TimeInterval;
+import com.google.monitoring.v3.TimeSeries;
+import com.google.protobuf.util.Timestamps;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import runner.config.ServerSpecification;
+
+public class MetricsUtils {
```

Comment:
`MetricsUtils` and `LogsUtils` have many similarities but enough differences that code sharing doesn't seem worth it. Too bad Google doesn't provide a single unified abstraction over both of these concepts to make your life easier!

Reply:
Agreed, more unified client library objects/structures would be much appreciated.